specs/experimental/cannon-fault-proof-vm-mt.md

```diff
@@ -252,11 +276,12 @@ The FPVM is a state transition function that operates on a state object consisti
 1. `preimageKey` - [`Hash`] The value of the last requested pre-image key.
 1. `preimageOffset` - [`Word`] The value of the last requested pre-image offset.
 1. `heap` - [`Word`] The base address of the most recent memory allocation via mmap.
-1. `llReservationActive` - [`Boolean`] Indicates whether a memory reservation,
-   which is reserved via a Load Linked Word (`ll`) instruction, is active.
+1. `llReservationStatus` - [`UInt8`] The current memory reservation status where: `0` means there is no
+   reservation, `1` means a 4-byte memory segment is reserved, and `2` means an 8-byte memory segment is reserved.
```

Comment:
The meaning of "4-byte memory segment" isn't clear from context. I suggest just using `ld/sc` and `lld/scd` as both semantics have already been defined on the previous section.